sc_rf_service/sc_rf_service.py

```diff
@@ -30,9 +56,14 @@ class HOMPVGroup(PVGroup):
 
 class HWIPVGroup(PVGroup):
     hwi = pvproperty(value=0, name="HWINITSUM", dtype=ChannelType.ENUM,
-                     enum_strings=("Ok", "HW Init running", "LLRF chassis problem"))
+                     enum_strings=("Ok", "HW Init running", "LLRF chassis problem"),
+                     record="mbbi")
```

Comment:
Out of curiosity, what is this `record="mbbi"` that has been added to a bunch of the PVs?

Reply:
Oh man I can't remember why I added them (I think I was trying to get waveforms to work lol) but MBBI are EPICS enums!